README.md

```diff
@@ -332,6 +332,19 @@ export irodsEnvFile=/path/to/.irodsEnv
 $DEST_BIN_DIR/testirodsmap "/C=XX/O=YYY/CN=Example User"
 ```
 
+Checksum Considerations
+-----------------------
+
+Checksums are calculated by doing a full open/read/close on the data object.  This plugin must read the file to calculate the checksum for the following reasons:
+
+1. There is no appropriate data object checksum API within iRODS so the client is responsible for calculating the checksum.
+2. The existing checksum functionality within iRODS doesn't support all of the desired checksum algorithms.
```

Comment:
I feel like these need to be combined, or reason 1 needs to be dropped. Reason 2 feels like the explanation for reason 1 to me. Thoughts?

Reply:
kind of...

1 is the lack of an appropriate singular API call.
1 is the lack of iRODS support for all of the Globus-supported algorithms.

Reply:
So `rcDataObjChksum` doesn't do what we want? Do we need to make a new API? I guess I'm left wanting a "why" for reason 1.

Reply:
I think it's because you can't select your checksum algorithm with `rcDataObjChksum`.

Reply:
looks better i think

Reply:
Yes, looks better.